Godeps/Godeps.json

```diff
@@ -1171,127 +1171,127 @@
 		},
 		{
 			"ImportPath": "github.com/gogo/protobuf/gogoproto",
-			"Comment": "v0.4-3-gc0656edd",
+			"Comment": "v0.4-3-gc0656ed",
```

Comment:
@vishh I'd like to avoid revendoring anything else than gRPC and it's dependecies (x/net and x/text).
I really think that doing so is outside the scope of this PR and would need to go through the whole re-testing on a 100 node cluster.

Let's keep the changes to the minimum to avoid breaking anything else :)